lux/processor/Compiler.py

```diff
@@ -395,9 +383,10 @@ def enforce_specified_channel(vis: Vis, auto_channel: Dict[str, str]):
             if len(sAttr) == 1:  # if specified in dobj
                 # remove the specified channel from auto_channel (matching by value, since channel key may not be same)
                 for i in list(auto_channel.keys()):
+                    # need to ensure that the channel is the same (edge case when duplicate Cols with same attribute name)
                     if (auto_channel[i].attribute == sAttr[0].attribute) and (
```

Comment:
Can we rewrite this to be inline?

Reply:
I wasn't able to make it inline, but I split it up so that the two conditions are on separate lines